Styr/styr v1.1

```diff
@@ -0,0 +1,799 @@
+/*
```

Comment:
Enda ändringen är kalibrering av hastighet och gyro-värden. La också in 10 ms i sensormodulen